pomegranate/gmm.pyx

```diff
@@ -167,17 +170,22 @@ cdef class GeneralMixtureModel(BayesModel):
 			else:
 				improvement = log_probability_sum - last_log_probability_sum
 
+				time_spent = time.time() - epoch_start_time
+				vals = dict(iteration=iteration, improvement=improvement, time=time_spent)
+				fmt = "[{iteration}] Improvement: {improvement} Time (s): {time:.2f}"
```

Comment:
I don't mind merging it now and then revisiting the issue later. Better to have the feature then clean it up.